README.md

```diff
@@ -36,9 +36,12 @@ We kindly ask all academic publications employing components of VILLASframework
 
 ## Contact
 
+[![EONERC ACS Logo](doc/pictures/eonerc_logo.png)](http://www.acs.eonerc.rwth-aachen.de)
+
 - Niklas Eiling <niklas.eiling@eonerc.rwth-aachen.de>
 - Steffen Vogel <post@steffenvogel.de>
 - Daniel Krebs <dkrebs@eonerc.rwth-aachen.de>
 
 [Institute for Automation of Complex Power Systems (ACS)](http://www.acs.eonerc.rwth-aachen.de)
+[EON Energy Research Center (EONERC)](http://www.eonerc.rwth-aachen.de)
```

Comment:
Why?`

Reply:
I would even remove all of these affiliations.

Reply:
I am fine with removing it.

But if we do, we should do it for all VILLAS sub-projects at once.
Otherwise the structure of all these READMEs diverges to much over time.

Reply:
Ok let's discuss at the next villas meeting. @iripiri